.circleci/config.yml

```diff
@@ -290,7 +290,7 @@ workflows:
       - production_deploy_approval:
           type: approval
           requires:
-            - Legacy staging deploy
+            - Helm Staging Deploy
           filters:
```

Comment:
I think that `name: Legacy staging deploy` should have the `requires staging_deploy_approval` removed. We want that to deploy asap with the new URL and therefore release the old URLs. 

And staging_deploy_approval should have a `requires` that checks that `Legacy staging deploy` is complete, that way we know we can release the helm staging deploy